docs/griptape-tools/official-tools/src/image_query_tool_1.py

```diff
@@ -1,13 +1,12 @@
-from griptape.drivers import OpenAiImageQueryDriver
+from griptape.drivers import OpenAiChatPromptDriver
 from griptape.structures import Agent
 from griptape.tools import ImageQueryTool
 
-# Create an Image Query Driver.
-driver = OpenAiImageQueryDriver(model="gpt-4o")
+driver = OpenAiChatPromptDriver(model="gpt-4o")
 
 # Create an Image Query Tool configured to use the engine.
 tool = ImageQueryTool(
```

Comment:
I'm curious about the use case when an `ImageQueryTool` is still useful? if the prompt is "describe this image", the model should be able to do that now without using a tool.

Reply:
There is still value in having a Tool that can query images from the file system/task memory. Long term this functionality should maybe be baked into the File Manager Tool but that would require refactors outside the scope of this PR.